### PR TITLE
Update asyncactioncompletedhandler.md

### PR DESCRIPTION
**Pull request by asklar**
IAsyncAction.Completed takes an IAsyncAction but the param name is asyncInfo

### CODE REVIEW

**Comment by PRMerger6** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by asklar** [comment]
@jwmsft will changing the signature in the xml comment automatically fix the type of the parameter and the link?

**Comment by asklar** [comment]
mmm the metadata says iasyncaction, but the cppwinrt projection says iasyncinfo... let me follow up with cppwinrt first before we merge this
CC @kennykerr

File: windows.foundation/asyncactioncompletedhandler.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 -api-type: winrt delegate
 ---
 <!-- Delegate syntax.
-public delegate void AsyncActionCompletedHandler(Windows.Foundation.IAsyncAction asyncInfo, Windows.Foundation.AsyncStatus asyncStatus)
+public delegate void AsyncActionCompletedHandler(Windows.Foundation.IAsyncInfo asyncInfo, Windows.Foundation.AsyncStatus asyncStatus)
```
Comment (asklar) [inline windows.foundation/asyncactioncompletedhandler.md]:
```suggestion
public delegate void AsyncActionCompletedHandler(Windows.Foundation.IAsyncAction asyncAction, Windows.Foundation.AsyncStatus asyncStatus)
```

**Comment by hickeys** [comment]
@asklar Any updates on this? Happy to move the conversation to email if you'd prefer - hickeys a microsoft

**Comment by asklar** [comment]
the type in the doc is correct, and the parameter name matches what's in the metadata, but the name in the metadata is wrong :)
I think it would be good to rename the parameter to `asyncAction`

**Comment by jwmsft** [comment]
@asklar, sorry I missed your question previously. The syntax that you updated in this PR is injected for the convenience of the author, but is not used in the published doc. What's published in the syntax block is generated entirely from the metadata and there's no way to manually update it.

**Pull request merged**